lib/core/topologies/server.js

```diff
@@ -55,6 +55,10 @@ var serverAccounting = false;
 var servers = {};
```

Comment:
This looks like it is based on #2191, correct?

Reply:
yep, rebased.